packages/visual-editor/src/components/atoms/cta.tsx

```diff
@@ -26,10 +32,52 @@ export const CTA = ({
   target,
   alwaysHideCaret,
   ariaLabel,
+  ctaType = "textAndLink",
+  coordinate,
+  presetImageType,
 }: CTAProps) => {
   let caretVisibility = "none";
+  let finalLink = link || "#";
+  let finalLinkType = linkType ?? "URL";
+  let finalLabel: React.ReactNode = label ?? "";
+  let finalAriaLabel = ariaLabel || "";
+
+  // Handle different CTA types
+  switch (ctaType) {
+    case "getDirections":
+      if (coordinate) {
+        const directionsLink = getDirections(
+          undefined,
+          undefined,
+          undefined,
+          { provider: "google" },
+          coordinate
+        );
+        finalLink = directionsLink || "#";
+        finalLinkType = "DRIVING_DIRECTIONS";
+        finalLabel = finalLabel || "Get Directions";
+        finalAriaLabel = finalAriaLabel || "Get Directions";
+      }
+      break;
+    case "presetImage":
+      if (presetImageType) {
+        finalLabel = presetImageIcons[presetImageType];
```

Comment:
Same as a previous comment. What if Consulting wanted to pass their own icon that isn't in our list? Where did our list even come from? It seems arbitrary.

Reply:
I'd start a thread in the project channel and ping Elisabeth, though I think she's OOO tomorrow